serviceregistration/consul/consul_service_registration.go

```diff
@@ -202,243 +198,142 @@ func NewConsulServiceRegistration(conf map[string]string, logger log.Logger) (sr
 	}
 
 	// Setup the backend
-	c := &ConsulServiceRegistration{
+	c := &ServiceRegistration{
 		Client: client,
 
 		logger:              logger,
+		state:               state,
 		serviceName:         service,
-		serviceTags:         strutil.ParseDedupLowercaseAndSortStrings(tags, ","),
+		usersTags:           strutil.ParseDedupLowercaseAndSortStrings(tags, ","),
```

Comment:
Should we continue to call it serviceTags?  That matches the `service_tags` name in the config file.

Reply:
Oh! Yes, definitely, I mainly changed it to keep it straight in my head where the tags came from.